src/components/FacetHeader.tsx

```diff
@@ -0,0 +1,89 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_faceting.scss';
+
+import { useRef, useState } from 'react';
+
+// Components
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+import DisplayValue from '@isrd-isi-edu/chaise/src/components/display-value';
+
+type FacetHeaderProps = {
+  key: string;
+  headerText: any;
+  showTooltipIcon: any;
+  tooltipContent: any;
+};
+
+/**
+ * @param key key prop to uniquely identify panel header item
```

Comment:
Move these comments to the custom type. Like we mentioned each property in the type should have its own comment in place.

Reply:
Resolved